### DISCUSSION TITLE
Create a separate leaderboard for NM and FM for Daily Challenges

### DISCUSSION THREAD

**Discussion by glaivelover972** [ppy/osu#29296] [Ideas]
Since the Daily Challenge is designed to be accessible for a majority of the community, it seems unfair that mods are allowed as part of the leaderboard. Players who can't play at that level with HDDT on every map don't have a chance to reach the top of the leaderboard anymore, and, at that point, they're only playing for a streak, not competition.

**Comment by kstefanowicz** [comment]
The very first Daily Challenge didn't have FM implemented, and it resulted in the Leaderboard being 50 of the exact same SS score:
![image](https://github.com/user-attachments/assets/fc8520cf-41fd-496d-8270-c3ef4dc64638)

So at least for the easier beatmaps, a NM leaderboard isn't super compelling for competition.

**Comment by peppy** [comment]
I'd sooner hide the leaderboard. The point of daily challenge isn't to get top place and maybe it's giving the wrong focus.

I don't think there's room in the system/screen to add mode filters to the leaderboard. I wanted to keep it simple..
